v2g-liberty-cards/src/ping-card.ts

```diff
@@ -79,30 +93,42 @@ export class PingCard extends LitElement {
     return this._isResponding
       ? nothing
       : html`
-          <ha-alert alert-type="error">
-            <div class="error">${tp('error')}</div>
-          </ha-alert>
-          <p>
-            <ha-markdown breaks .content=${tp('error-subtext')}></ha-markdown>
-          </p>
-          <mwc-button @click=${this._restart}>${tp('restart')}</mwc-button>
+          <mwc-snackbar
+            ?open=${this._snackbarOpen}
+            labelText=${this._isRestarting ? tp('restarting') : tp('error')}
+            timeoutMs="-1"  <!-- Set timeout to -1 for persistent behavior -->
+            persistent  <!-- Ensure it remains visible until manually closed -->
+            @closed=${this._onSnackbarClosed}
+          >
+            <mwc-button slot="action" @click=${this._restart}>${tp('restart')}</mwc-button>
```

Comment:
After hitting "Restart" the text changes to "Restarting..." but the button stays visible and can be used again.  I propose to remove the button while it is restarting, otherwise people may end up restarting the add-on multiple times.
Also I propose to change the restarting text to:
_Connection with V2G Liberty lost. Restarting..._

Reply:
Good suggestion! Will do.